README.md

```diff
@@ -1,7 +1,39 @@
 
 # hec-stubs
 
-This is a placeholder README.md for a new repository
+# Test Data
```

Comment:
I thought it'd be handy to document everything that's easy to see in one place as soon you go to this stub repo in github. With this collected info in this README I was thinking we could probably do without the commented out docs in the controllers and other places describing the same thing - what do you think @priyankadhiman3006  @meghana-viswanath? Save us updating the docs in multiple places if we need to change something in the future.

Reply:
good idea 👍